include/amici/model.h

```diff
@@ -202,9 +203,11 @@ class Model : public AbstractModel, public ModelDimensions {
      * @param sdx Reference to time derivative of state sensitivities (DAE only)
      * @param computeSensitivities Flag indicating whether sensitivities are to
      * be computed
+     * @param roots_found boolean indicators indicating whether roots were found at t0 by this fun
```

Comment:
current `int`, but okay, I guess.

Reply:
yes, more convenient to store as int due to data structure expected by sundials